### DISCUSSION TITLE
Images Dimensions Problem

### DISCUSSION THREAD

**Discussion by Bader-Hasan** [sumatrapdfreader/sumatrapdf#2715] [Q&A]
I have a folder with a list of images, and occasionally while scrolling I'll find an image that exceeds the width of my screen. 

My question: Is there a way to fit "ONLY" larger images either by width or content options instead of manually doing it each time.
I don't want to fit every image because the other ones will look small.

My settings (if it matters):
View:
Show pages continuously 
single page

Zoom: 100% (Actual Size)

**Comment by GitHubRulesOK** [comment]
zoom is probably over-riding view
here we see different shapes are adjusted to current screen height single pages continuous, thus both are shown as `Fit Page` not 100 %
![image](https://user-images.githubusercontent.com/12754913/173207905-c2a74583-3219-424e-8fa9-50cad3c577f7.png)


here they are at 100%

![image](https://user-images.githubusercontent.com/12754913/173208197-d059f3aa-ef23-4e98-a115-bb5b713e798e.png) or 
![image](https://user-images.githubusercontent.com/12754913/173208244-d90b8f31-9e65-48ba-b796-7fad3adb7fc8.png)

**Comment by Bader-Hasan** [comment]
Hmm... You seem to be right. 
The issue wasn't with my view settings. I changed the zoom to 90% and it did fix it.  
Although I would've wished for a solution similar to what's in (Irfanview) 
where you have the option to fit "only" larger images, I'm going to stick with this for the time being.

Thanks anyway!
